### PR TITLE
Edit data dource and panel READMEs

### PR DESCRIPTION
**Pull request by josmperez**
Edit several READMEs for Grafana style and readability. This PR will be followed by additional PRs to enhance the examples docs.

### CODE REVIEW

File: examples/datasource-streaming-backend-websocket/README.md
Comment (xnyo) [inline examples/datasource-streaming-backend-websocket/README.md:1]: For consistency, shall we change:

```
Build the data source plugin
```

:point_down: 

```
Build the data source plugin:
```

File: examples/datasource-basic/README.md
Comment (xnyo) [inline examples/datasource-basic/README.md:1]: For consistency, shall we add colons at the end of each step in the "Frontend" section. For example:

```
1. Install dependencies
```

:point_down: 

```
1. Install dependencies:
```

**Review by xnyo — COMMENTED** [review]
Awesome work! Some minor comments:

File: examples/datasource-basic/README.md
Comment (josmperez) [reply examples/datasource-basic/README.md:1]: done

File: examples/datasource-streaming-backend-websocket/README.md
Comment (josmperez) [reply examples/datasource-streaming-backend-websocket/README.md:1]: done

**Review by xnyo — APPROVED** [review]
LGTM! :rocket:

**Review by academo — APPROVED** [review]
All looks correct. But as mentioned in a similar PR. we are not using `yarn` for any of our examples. It'd be great to replace all calls to yarn with `npm run`.

e.g `yarn dev` should be `npm run dev` except for install which should remain simply `npm install`

File: examples/datasource-http-backend/README.md [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
-This differs from the datasource-http example, because the data fetching happens in the plugin backend rather than going through Grafana's datasource HTTP proxy.
+This differs from the `datasource-http` example because the data fetching happens in the plugin backend rather than going through Grafana's data source HTTP proxy.
 
-This allows to use the data source for alerting as well, as the query are executed on the backend.
+This allows the plugin to use the data source for alerting as well, as the queries are executed on the backend.
 
 This plugin example also showcases other features and best-practices of backend plugins:
 
-- Using the httpclient provided by the Grafana Plugins SDK
+- Using the `httpclient` provided by the Grafana plugins SDK
```
Comment (sympatheticmoose) [inline examples/datasource-http-backend/README.md:11]:
```suggestion
- Using the `httpclient` provided by the [Grafana plugins SDK](https://pkg.go.dev/github.com/grafana/grafana-plugin-sdk-go/backend/httpclient)
```

non blocking - could optionally add a link to the package or the SDK in general

**Review by sympatheticmoose — APPROVED** [review]
changes look good to me 👍 agree with esteban that we should update from Yarn but could be done in separate PR by anyone

**Pull request merged**